tests/kernel/mem_pool/mem_pool_api/src/main.c

```diff
@@ -19,6 +19,7 @@ extern void test_mpool_alloc_free_isr(void);
 extern void test_mpool_kdefine_extern(void);
 extern void test_mpool_alloc_size(void);
 extern void test_mpool_alloc_timeout(void);
+extern void test_sys_mem_pool_assign(void);
```

Comment:
change naming, this is to be confused with sys_mem_pool which is in another test.

Reply:
@nashif Changed the test name to test_sys_heap_mem_pool_assign.